src/orange/construct/detail/PostfixLogicBuilder.hh

```diff
@@ -0,0 +1,149 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2023 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file orange/construct/detail/PostfixLogicBuilder.hh
+//---------------------------------------------------------------------------//
+#pragma once
+
+#include "corecel/cont/VariantUtils.hh"
+#include "orange/OrangeTypes.hh"
+
+#include "../CsgTree.hh"
+
+namespace celeritas
+{
+namespace csg
+{
+//---------------------------------------------------------------------------//
+/*!
+ * Recursively construct a logic vector from a node with postfix operation.
```

Comment:
This class could use some additional high-level documentation. While I that how the `ContainerVisitor` is calling the operator() functions, and that how `PostfixLogicBuilder::operator()(NodeId const& n)` provides recursion, this is pretty hard to follow.